csharp/ql/src/utils/modelgenerator/internal/CaptureSummaryFlowQuery.qll

```diff
@@ -89,5 +89,5 @@ string captureFlow(DataFlowSummaryTargetApi api) {
 string captureNoFlow(DataFlowSummaryTargetApi api) {
   not exists(DataFlowSummaryTargetApi api0 | exists(captureFlow(api0)) and api0.lift() = api.lift()) and
   api.isRelevant() and
-  result = Printing::asNeutralSummaryModel(api)
+  result = ModelPrinting::asNeutralSummaryModel(api)
```

Comment:
It looks like the contents of this query can also be fully shared?

Reply:
The historical reason for not doing so, is because of the language specific examples in the comments. Maybe we should just consider scrapping those as this is now captured in a more readable way in the tests.